Documentation/security/network/encryption-wireguard.rst

```diff
@@ -295,6 +295,45 @@ options:
   - LoadBalancer & NodePort XDP Acceleration
   - Direct Server Return (DSR)
 
+Which traffic encrypted
+=======================
+
+The following table denotes which packets are encrypted with WireGuard depending
+on the mode:
+
++------------------------------------+--------------------+-------------------------+
+| Traffic                            | default mode       | node-to-node encryption |
+|                                    |                    | mode                    |
++------------------------------------+--------------------+-------------------------+
+| Pod to remote Pod                  | Y                  | Y                       |
++------------------------------------+--------------------+-------------------------+
+| Pod to remote Pod via L7 proxy or  | Y                  | Y                       |
+| L7 ingress service                 |                    |                         |
++------------------------------------+--------------------+-------------------------+
+| Pod to remote Pod via Service with | Y                  | Y                       |
+| kube-proxy-replacement             |                    |                         |
++------------------------------------+--------------------+-------------------------+
+| Pod to Egress Gateway node         | Y                  | Y                       |
++------------------------------------+--------------------+-------------------------+
+| Client outside cluster to Pod via  | Y                  | Y                       |
+| ServiceService with tunneling and  |                    |                         |
```

Comment:
```suggestion
| Service with tunneling and  |                    |                         |
```

Reply:
So the traffic is "obviously" not encrypted until it hits the Cilium node. The "Y" here only refers to traffic forwarded between Cilium-managed nodes. 

This seems clear to us engineers, but I wonder if we should call this out explicitly for someone who hasn't thought about the fact that traffic coming from outside might be forwarded between nodes.